### ISSUE TITLE
waypoint config panics

### ISSUE THREAD

**Issue by remyleone** [hashicorp/waypoint#520]
**Describe the bug**

```
$ waypoint config
panic: flag redefined: app

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0003f24e0, 0x46c7740, 0xc000b56160, 0x40138cc, 0x3, 0x4084216, 0x26)
	/usr/local/go/src/flag/flag.go:851 +0x4b8
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).Var(0xc0003ea700, 0x46c7740, 0xc000b56160, 0x40138cc, 0x3, 0x4084216, 0x26)
	/home/circleci/project/waypoint/internal/pkg/flag/flag_var.go:79 +0x6f
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).VarFlag(0xc0003ea700, 0xc000acaf00)
	/home/circleci/project/waypoint/internal/pkg/flag/flag_var.go:72 +0x1e1
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).StringVar(0xc0003ea700, 0xc000acae00)
	/home/circleci/project/waypoint/internal/pkg/flag/flag_string.go:33 +0x212
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Flags.func1(0xc000b76180)
	/home/circleci/project/waypoint/internal/cli/config_get.go:144 +0x18a
github.com/hashicorp/waypoint/internal/cli.(*baseCommand).flagSet(0xc0003dc000, 0x0, 0xc000c1fb40, 0x101309b)
	/home/circleci/project/waypoint/internal/cli/base.go:415 +0x224
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Flags(0xc000b56080, 0xc000c1fb98)
	/home/circleci/project/waypoint/internal/cli/config_get.go:130 +0x5a
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Help(0xc000b56080, 0xc000b76150, 0x4025d07)
	/home/circleci/project/waypoint/internal/cli/config_get.go:171 +0x2f
github.com/mitchellh/cli.(*CLI).commandHelp(0xc0003dc140, 0x46a3fc0, 0xc00000e020, 0x46f8840, 0xc000b0d3c0)
	/go/pkg/mod/github.com/mitchellh/cli@v1.1.2/cli.go:575 +0x894
github.com/mitchellh/cli.(*CLI).Run(0xc0003dc140, 0xc0003dc140, 0xc000b0c000, 0xc0009a1f10)
	/go/pkg/mod/github.com/mitchellh/cli@v1.1.2/cli.go:265 +0x227
github.com/hashicorp/waypoint/internal/cli.Main(0xc000a43700, 0x2, 0x2, 0x0)
	/home/circleci/project/waypoint/internal/cli/main.go:97 +0x4ee
main.main()
	/home/circleci/project/waypoint/cmd/waypoint/main.go:14 +0xa2

```

**Steps to Reproduce**

`waypoint config`

**Expected behavior**

waypoint not panicking

**Additional context**

**Comment by tpetry** [comment]
Happening for me too. I am running `Waypoint v0.1.1 (44e2d3d6)` on MacOS.

**Comment by rugwirobaker** [comment]
@remyleone I raised the same issue area but was told it has already been solved in main. A fix is coming in the next release

**Comment by rvbuelow** [comment]
Fixed in #473

**Comment by pearkes** [comment]
Thanks @rugwirobaker and @rvbuelow, this is indeed fixed in #473.
